experimental/saiexperimentaldash.h

```diff
@@ -205,6 +205,72 @@ typedef enum _sai_eni_attr_t
      */
     SAI_ENI_ATTR_ADMIN_STATE,
 
+    /**
+     * @brief Action set_eni_attrs parameter INBOUND_STAGE1_DASH_ACL_GROUP_ID
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_DASH_ACL_GROUP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_ENI_ATTR_INBOUND_STAGE1_DASH_ACL_GROUP_ID,
```

Comment:
We can have STAGE1_ACL_V4 and V6 separate

Reply:
Done